app/src/common/css/variables/z-index-variables.scss

```
@@ -20,8 +20,11 @@ $Z_INDEX-STICKY-TOOLBAR: 3;
 $Z_INDEX-USER_BLOCK_DROPDOWN: 4;
 $Z_INDEX-INPUT_DROPDOWN: 4;
 $Z_INDEX-ALL_LATEST_DROPDOWN: 4;
+$Z-INDEX-DARK-MODAL: 5;
 $Z-INDEX-POPUP: 10;
 $Z-INDEX-PAGE-CONTAINER-HEADER: 11;
 $Z-INDEX-HINT: 15;
 $Z-INDEX_AUTOCOMPLETE: 16;
 $Z-INDEX-OVERLAY-PANEL: 100;
+// Topmost app overlay — must cover modals, popups, and side panels during async operations.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)